wrappers/windef.go

```diff
@@ -16,10 +16,16 @@
 
 package wrappers
 
+import "syscall"
+
 const (
 	MAX_PATH = 260
 )
 
+type HDC syscall.Handle
+
+type HMONITOR syscall.Handle
```

Comment:
We generally don't define type aliases for Win32 types.  Instead, we can just use `syscall.Handle`.